server/org.eclipse.emfcloud.ecore.glsp/pom.xml

```diff
@@ -9,7 +9,7 @@
 		<repository>
 			<id>sonatype</id>
 			<name>Sonatype</name>
-			<url>https://oss.sonatype.org/content/groups/public</url>
```

Comment:
`groups/public` seems to be the correct repository to download Releases and Snapshots. Is there any reason why you'd prefer `repositories/snapshots`?

Reply:
It was 404 when I tried to build. The change fixed it. Do you have a different URL I can change it to that works?

Reply:
@CamilleLetavernier  https://oss.sonatype.org/content/groups/public is still `404`. Is this just intermittent you think?

Reply:
The page is 404 when you access it from a web browser, but it's still a valid maven repository; I don't have any issue building the project with this repository:

```
Downloading from sonatype: https://oss.sonatype.org/content/groups/public/org/eclipse/glsp/org.eclipse.glsp.layout/0.8.0-SNAPSHOT/maven-metadata.xml
Downloaded from sonatype: https://oss.sonatype.org/content/groups/public/org/eclipse/glsp/org.eclipse.glsp.layout/0.8.0-SNAPSHOT/maven-metadata.xml (1.0 kB at 5.4 kB/s)
```

Reply:
Oh, that's not good (for me)! I will give it another try with the original repo. Maybe it was just an intermittent network error.